docs/contributing/coding-guidelines.md

```diff
@@ -18,82 +15,76 @@ $ ng lint
-
-All that is required is to install the TSLint plugin. So far, it works out of the box without the need to add further configuration. (TSlint reads the project's `tslint.json` file and seems to figure it out on its own.)
+To make the standards adoption even more seamless, Visual Studio Code (VS Code) and other editors can provide live feedback from the rules reported by `ng lint`.
 
-### Coding guidelines in DOD
+We use a number of VS Code extensions for code compliance. When you open the source folder in VS Code, if you are missing any of the recommended extensions, VS Code prompts you to install them.
 
-Use the TSLint vscode plugin. Code styling errors will show up in red as you type.
+You can see the list of recommended extensions in `.vscode/extensions.json`.
 
+If you are missing any of the recommended extensions, please install them.
 
-
-## Spartacus Specific Guidelines
+## Spartacus-Specific Guidelines
 
 ### NGRX
 
-We have chosen to use the NGRX store to manage the global application state in our features. Using NGRX has apparent advantages for performance, better testability, and ease of troubleshooting (with time travel and such).
+We use the NGRX store to manage the global application state in our features. Using NGRX has apparent advantages for performance, better testability, and ease of troubleshooting (with time travel and such).
 
-- The rule of thumb is to use the store for a feature unless there is a compelling reason not to.  We want to keep it consistent throughout the app.
+- Use the store for a feature unless there is a compelling reason not to. We want to keep it consistent throughout the app.
 - Use one common store for the whole app.
-- Using the store doesn’t mean that we need to cache everything.  Caching should be used with intent and where it makes sense.  In general, cms data is a good candidate for caching; application data is not.
+
+**Note**: Using the store does not mean that we need to cache everything. Caching should be used with intent, and where it makes sense. In general, CMS data is a good candidate for caching, while application data is not.
 
 ### Site Context
 
-Site context can be changed for each page. The response data may be different for different site context. Keep this in mind when working on pages.
+Site context can be changed for each page. The response data may be different for different site contexts. Keep this in mind when working on pages.
 
 Also, logged-in users and anonymous users may see different response data. When working on pages, take into consideration that a user can change their login status through login or logout.
 
 ### Naming Conventions
 
-Component selectors should always start with "cx-". (Currently, many component selectors start with "y-" but this is being chnaged.)
-
-### Code Structure
+Component selectors should always start with "cx-".
```

Comment:
Very good point! I have modified it to read as follows:

"If you work for a team inside SAP, then component selectors should always start with `cx-` (such as `cx-banner`, for example). If you are a partner or customer, you should use a different prefix (that is, something other than `cx-`) to avoid conflicts with Spartacus libraries, and third-party libraries that are used by Spartacus."